<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CH</class>
 <widget class="QMainWindow" name="CH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CH</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion Archivo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Estatus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>561</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Empecemos!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LlaveEmisorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione llave </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion de la llave del Emisor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Criptograf√≠a Hibrida</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="EmisorRadioButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>124</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Emisor</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LlaveEmisorPath">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="ReceptorRadioButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Receptor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ArchivoButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione Archivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>79</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Estatus:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="archivoPath">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>560</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LlaveReceptorPath">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LlaveReceptorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione llave </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>281</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion de la llave del Receptor:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QLabel" name="llaveEmisorLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="llaveReceptorLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
